packs/slack/sensors/slack_sensor.py

```diff
@@ -179,7 +180,8 @@ def _handle_message(self, data):
                 'topic': channel_info['topic']['value'],
                 'is_group': channel_info.get('is_group', False),
             },
-            'timestamp': int(float(data['ts'])),
+            'timestamp': int(string.replace(data['ts'], '.', ''))),
```

Comment:
So this value is now representing microseconds since the epoch?

If so, we should probably add a new attribute since timestamp usually refers to a unix epoch in seconds.